pkgs/development/tools/sqlelf/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+
+# build-system
+, setuptools-scm
+
+# dependencies
+, capstone
+, lief
+, apsw
+, sh
+
+# tests
+, pytestCheckHook
+, coreutils
+}:
+
+buildPythonPackage rec {
+  pname = "sqlelf";
+  version = "0.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-S2D918B/zuIpoF/FBTQKMjmPLjXt11wnxXMQ5rZrp7s=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  nativeBuildInputs = with python3Packages; [
```

This is what I meant. Same for `propagatedBuildInputs` and `nativeCheckInputs`.